src/main/java/pl/tomaszdziurko/jvm_bloggers/blog_posts/RssCheckingActor.java

```diff
@@ -32,17 +31,13 @@ public static Props props(ActorRef postStoringActor, SyndFeedProducer syndFeedFa
     @SuppressWarnings("unchecked")
```

Comment:
This warning suppression is not necessary. There is no unchecked operation here.